keyboards/gmmk/pro/keymaps/alexmarmon/rules.mk

```diff
@@ -0,0 +1,2 @@
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = AW20216
```

Comment:
https://github.com/qmk/qmk_firmware/blob/40d13e1210afc65f01f9afc4fa68086892862210/keyboards/gmmk/pro/rules.mk#L24-L25

Already specified at the parent level so not required. As there are no other overrides, this file should be removed/